compiler/util/files.cpp

```diff
@@ -424,13 +425,13 @@ const char* createDebuggerFile(const char* debugger, int argc, char* argv[]) {
   return dbgfilename;
 }
 
-const std::string runUtilScript(const char* script) {
+std::string runUtilScript(const std::string script) {
```

Comment:
It doesn't usually matter in any specific situation but it's general recommended to prefer "const std::string& script" to "const std::string script".